File: app/src/main/java/com/yhjoo/dochef/classes/Comment.java
package com.yhjoo.dochef.classes;

import com.google.gson.annotations.SerializedName;

public class Comment {
    @SerializedName("NICKNAME")
    private final String nickName;
    @SerializedName("COMMENT")
    private final String contents;
    @SerializedName("TIME")
    private final String dateTime;
    @SerializedName("USER_ID")
    private final String userID;
    @SerializedName("COMMENT_ID")
    private int CommentID;
    @SerializedName("RECIPE_ID")
    private int ReciepeID;

    public Comment(String userID, String nickName, String contents, String dateTime) {
        this.userID = userID;
        this.nickName = nickName;
        this.contents = contents;
        this.dateTime = dateTime;
    }

    public String getUserID() {
        return userID;
    }

    public String getNickName() {
        return nickName;
    }

    public String getContents() {
        return contents;
    }

    public String getDateTime() {
        return dateTime;
    }
}

File: app/src/main/java/com/yhjoo/dochef/data/model/Comment.kt
package com.yhjoo.dochef.data.model

import com.google.gson.annotations.SerializedName

data class Comment(
    @SerializedName("comment_id") val commentID: Int,
    @SerializedName("post_id") val postID: Int,
    @SerializedName("user_id") val userID: String,
    @SerializedName("nickname") val nickName: String,
    @SerializedName("user_profile_img") val userImg: String,
    @SerializedName("contents") val contents: String,
    @SerializedName("datetime") val dateTime: Long
)